### ISSUE TITLE
Add "how we work together" docs

### ISSUE THREAD

**Issue by mfisher87** [earthaccess-dev/earthaccess#529]
Should be part of the contributing docs. Link out to openscapes team culture documentation, talks, etc.

Related: #514 #525 #521

EDIT: Also #537 . I feel there's an obligation on maintainers to maintain team culture as well (not in terms of time obligations like giving presentations, but in terms of applying that culture to their other actions as maintainers) that we need to make explicit in that document, but we should probably think about these two tickets as a whole together.

**Comment by Sherwin-14** [comment]
Hey, should I proceed forward in this one? do you have any opinions about this?

**Comment by mfisher87** [comment]
@asteiker thoughts? With the new maintainer doc page @Sherwin-14 contributed recently, what is still missing? :thinking:

**Comment by Sherwin-14** [comment]
> @asteiker thoughts? With the new maintainer doc page @Sherwin-14 contributed recently, what is still missing? 🤔

Any updates on this one? I feel this is quite similar to previous ones so it would be easier for us to tackle this.

**Comment by saberbrasher** [comment]
I believe this issue can be closed; 
https://earthaccess.readthedocs.io/en/stable/contributing/code-of-conduct/
and
https://earthaccess.readthedocs.io/en/stable/contributing/maintainers-guide/
seem to address these wants. Tagging @asteiker for confirmation.

**Comment by asteiker** [comment]
@Sherwin-14 I agree with @saberbrasher here that we have comprehensive documentation now on how to work with us. Starting with the top-level https://earthaccess.readthedocs.io/en/stable/contributing/, we now have specific guides on contributing, triaging, the maintainer's guide, and meet-ups. 

The [Birdaro training](https://www.cscce.org/2025/07/01/the-birdaro-training-program-supporting-open-source-leaders/) is entering its second phase with a focus on Community Playbooks and I am planning to analyze this section in the context of the training. So, I may write up new issues or discussions as I learn more but for now I'd say this could be closed out.
